notes/stv/20200409-issue-33-ci-actions.txt

```diff
@@ -0,0 +1,114 @@
+#
+# <meta:header>
+#   <meta:licence>
+#     Copyright (c) 2019, ROE (http://www.roe.ac.uk/)
+#
+#     This information is free software: you can redistribute it and/or modify
+#     it under the terms of the GNU General Public License as published by
+#     the Free Software Foundation, either version 3 of the License, or
+#     (at your option) any later version.
+#
+#     This information is distributed in the hope that it will be useful,
+#     but WITHOUT ANY WARRANTY; without even the implied warranty of
+#     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#     GNU General Public License for more details.
+#  
+#     You should have received a copy of the GNU General Public License
+#     along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#   </meta:licence>
+# </meta:header>
+#
+#
+
+
+# ------------------------------------------------------
+# Creating a CI workflow with Github Actions
+# ------------------------------------------------------
+
+# The following describes how to create a Github Continuous integration worfklow, to trigger a set of unit tests
+# It assumes a created Dockerfile and set of unit tests included in the project directory
+# In the following example the Dockerfile looks like this:
+
+
```

Comment:
Does the Docker file need to be in a specific location ?
Can we add a config param to point to a different Dockerfile ?